### ISSUE TITLE
Exception if Aads/VariousControls/MiscControls is empty

### ISSUE THREAD

**Issue by escherstair** [bojanpotocnik/cmake-uvision-syncer#4]
I noticed that exception _'NoneType' object has no attribute 'split'_ comes out if `Aads/VariousControls/MiscControls` is empty.
Line
https://github.com/bojanpotocnik/cmake-uvision-syncer/blob/049524cc18bcd17e2db54dee6b97d69429e37c5b/main.py#L414
calls `text(...).split()` but `text()` can be `NoneType` if `Aads/VariousControls/MiscControls` is empty.
Maybe the same exception can happen somewhere else.

**Comment by Rian-Night** [comment]
yes, it didnt work when contents are empty.  LDads is also called wrong
